app/javascript/v3/helpers/CommonHelper.js

```diff
@@ -1,3 +1,9 @@
 export const replaceRouteWithReload = url => {
   window.location = url;
 };
+
+export const userInitial = name => {
```

Comment:
Let us add a case to prevent a case when the name is undefined. `if (!name) return ''`